documents/Specification/MaterialX.StandardNodes.md

```diff
@@ -148,22 +148,22 @@ The `file` input can include one or more substitutions to change the file name t
-|`out`               |Output: the sampled texture value                                                                  |Same as `default`     |__zero__                       |
+|Port                |Description                                                                                        |Type             |Default                        |
+|--------------------|---------------------------------------------------------------------------------------------------|-----------------|-------------------------------|
+|`file`              |The URI of the image file                                                                          |filename         |__empty__                      |
+|`default`           |A default value to use if the file reference can not be resolved                                   |colorN           |__zero__                       |
+|`texcoord`          |The 2D texture coordinate at which the image data is read                                          |vector2          |_UV0_                          |
+|`tiling`            |The tiling rate for the given image along the U and V axes                                         |vector2          |1.0, 1.0                       |
+|`rotation`          |Per-tile rotation randomness in degrees                                                            |float            |0.0                            |
+|`rotationrange`     |Range in degrees used to randomize rotation for each tile                                          |vector2          |0.0, 360.0                     |
+|`scale`             |Per-tile scale randomness multiplier applied to tile size                                          |float            |1.0                            |
+|`scalerange`        |Range of scale multipliers used to randomize tile scale                                            |vector2          |0.5, 2.0                       |
+|`offset`            |Per-tile translation randomness in UV units                                                        |float            |1.0                            |
+|`offsetrange`       |Range of offset values in UV units used to randomize tile positions                                |vector2          |0.0, 1.0                       |
+|`falloff`           |Falloff width used to blend neighboring tiles at their edges; larger values produce smoother blends|float            |0.5                            |
+|`falloffcontrast`   |Contrast applied to the falloff blending to sharpen (values >1) or soften (values <1) transitions  |float            |0.5                            |
+|`lumacoeffs`        |The luma coefficients of the current working color space                                           |color3           |0.2722287, 0.6740818, 0.0536895|
+|`out`               |Output: the sampled texture value                                                                  |Same as `default`|__zero__                       |
```

Comment:
Nitpick: For consistency with the other nodes would it make sense to have the output define the data type, and let `default` derive from this? That is, just flip the relation here so `default` says _Same as 'out'_

Reply:
Good idea, and I've made this change across the texture nodes for consistency.